python-net/advanced-operations/working-with-forms/acroforms/_index.md

```diff
@@ -0,0 +1,157 @@
+    priority: 0.7
+---
+<script type="application/ld+json">
+{
+    "@context": "https://schema.org",
+    "@type": "TechArticle",
+    "headline": "Working with AcroForms using Python",
+    "alternativeHeadline": "Options for working with AcroForms in PDF",
+    "author": {
+        "@type": "Person",
+        "name":"Anastasiia Holub",
+        "givenName": "Anastasiia",
+        "familyName": "Holub",
+        "url":"https://www.linkedin.com/in/anastasiia-holub-750430225/"
+    },
+    "genre": "pdf document generation",
+    "keywords": "pdf, python, acroforms in pdf",
+    "wordcount": "302",
+    "proficiencyLevel":"Beginner",
+    "publisher": {
+        "@type": "Organization",
+        "name": "Aspose.PDF Doc Team",
+        "url": "https://products.aspose.com/pdf",
+        "logo": "https://www.aspose.cloud/templates/aspose/img/products/pdf/aspose_pdf-for-python-net.svg",
+        "alternateName": "Aspose",
+        "sameAs": [
+            "https://facebook.com/aspose.pdf/",
+            "https://twitter.com/asposepdf",
+            "https://www.youtube.com/channel/UCmV9sEg_QWYPi6BJJs7ELOg/featured",
+            "https://www.linkedin.com/company/aspose",
+            "https://stackoverflow.com/questions/tagged/aspose",
+            "https://aspose.quora.com/",
+            "https://aspose.github.io/"
+        ],
+        "contactPoint": [
+            {
+                "@type": "ContactPoint",
+                "telephone": "+1 903 306 1676",
+                "contactType": "sales",
+                "areaServed": "US",
+                "availableLanguage": "en"
+            },
+            {
+                "@type": "ContactPoint",
+                "telephone": "+44 141 628 8900",
+                "contactType": "sales",
+                "areaServed": "GB",
+                "availableLanguage": "en"
+            },
+            {
+                "@type": "ContactPoint",
+                "telephone": "+61 2 8006 6987",
+                "contactType": "sales",
+                "areaServed": "AU",
+                "availableLanguage": "en"
+            }
+        ]
+    },
+    "url": "/python-net/acroforms/",
+    "mainEntityOfPage": {
+        "@type": "WebPage",
+        "@id": "/python-net/acroforms/"
+    },
+    "dateModified": "2023-02-04",
+    "description": "With Aspose.PDF for Python you may create a form from scratch, fill the form field in a PDF document, extract data from the form, and etc."
+}
+</script>
+
+## Fundamentals of AcroForms
+
+**AcroForms** are the original PDF forms technology. AcroForms is a page oriented form. They were first introduced in 1998. They accept input in Forms Data Format or FDF and XML Forms Data Format or xFDF. Third party vendors support AcroForms. When Adobe introduced the AcroForms, they referred to them as “PDF form that is authored with Adobe Acrobat Pro/Standard and that is not a special type of static or dynamic XFA form. Acroforms are portable and they work on all platforms.
```

Comment:
**AcroForms** - unique PDF form technology from Adobe. AcroForms is a page-oriented form. They first appeared in 1998. They accept input in the form of Data format or FDF and XML form Data format or xFDF. Third-party providers support AcroForms. When Adobe introduced AcroForms, they called them the "PDF form, which is the author of Adobe Acrobat Pro/Standard and is not a special type of static or dynamic XFA form. Acroforms are portable and work on all platforms.
```suggestion
**AcroForms** - unique PDF form technology from Adobe. AcroForms is a page-oriented form. They first appeared in 1998. They accept input in the form of Data format or FDF and XML form Data format or xFDF. Third-party providers support AcroForms. When Adobe introduced AcroForms, they called them the "PDF form, which is the author of Adobe Acrobat Pro/Standard and is not a special type of static or dynamic XFA form. AcroForms are portable and work on all platforms.```